docs/articles/azure/azs-how-create-key-vault-ps.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to create a Key Vault - PowerShell | UKCloud Ltd
+description: Provides help for creating a Key Vault on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Key Vault - PowerShell
+toc_fullpath: Users/How To/azs-how-create-key-vault-ps.md
+toc_mdlink: azs-how-create-key-vault-ps.md
+---
+
+# How to create a Key Vault in Azure Stack with PowerShell
+
+## Overview
+
+Key Vault in Azure Stack helps safeguard cryptographic keys and secrets that cloud applications and services use. By using Key Vault, you can encrypt keys and secrets.
+
+The following process shows you how to setup a vault within Azure Stack's Key Vault, store a secret in the vault and how to extract the secret.
+
+## Prerequisites
+
+Ensure your PowerShell environment is setup as detailed in [Configure the Azure Stack user's PowerShell environment](azs-how-configure-powershell-users.html).

```

Comment:
Sorry, change file name in the link to azs-how-configure-powershell-users.md

(i.e., point to the markdown file, not the html)

Reply:
Fixed